rest_api/rest_api/schema.py

```diff
@@ -22,9 +22,20 @@
 PrimitiveType = Union[str, int, float, bool]
 
 
+class PipelineHyperParams(BaseModel):
+    faq_embedding_dim: int = 384
+    extractive_embedding_dim: int = 768
+    extractive_reader_option: str = "deepset/roberta-base-squad2"
+    faq_retriever_option: str = "sentence-transformers/all-MiniLM-L6-v2"
+    faq_similarity_function: str = "dot_product"
```

Comment:
same as in `configuration.json` file - faq similarity function should be cosine, and extractive can be dot_product :)

Reply:
https://github.com/rafaljanwojcik/board-game-rules-explainer/pull/4/commits/d5847c61d41f6690162a7e5b728e447827a162f2